initial_bot/mentor_bot_test.py

```diff
@@ -19,85 +61,97 @@ def parse_arguments():
                         default="mentorbot_test")
     return parser.parse_args()
 
+intents = discord.Intents.all()
+bot = commands.Bot(command_prefix = '!', intents = intents, case_insensitive=True)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
maybe move the logger initialization to `main()`, so it doesn't happen when this file is imported as a library?